packages/cli/src/commands/onboard/README.md

```diff
@@ -0,0 +1,23 @@
+# admin tool
+
+## Local dev
+
+```bash
+# Checkout the branch
+cd backstage
+git fetch
+git checkout goa/cli-admin-command
+
+# Create a new app
+cd ..
+npx @backstage/create-app
```

Comment:
With `--skip-install` if you wanna make it quick 😁, don't think you need deps for testing this new command right?

Reply:
yeah - don't really need deps for the command. It could be annoying afterwards though, as you will probably want to run it directly after the `onboard` command & then have to wait for the install. Either way is totally fine for me! :)